src/ctia/domain/entities.clj

```diff
@@ -88,11 +88,8 @@
     (long-id->entity-type id-str)))
 
 (defn un-store [record]
-  (let [entity-fields [:created :modified :owner :groups]
-        event-fields [:created :modified]]
-    (case (:type record)
-      "event" (apply dissoc record event-fields)
-      (apply dissoc record entity-fields))))
+  (let [entity-fields [:created :modified]]
+    (apply dissoc record entity-fields)))
```

Comment:
that list is short and you now have only one case, you should directly shorten it to: 
``` clojure
(apply dissoc record [:created :modified])
```